tests/UI/scripts/generate-jsdoc.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# To print only errors
+set -x
+
+# Get tests/UI directory path
+PROJECT_PATH=$(cd "$( dirname "$0" )/../" && pwd)
+cd $PROJECT_PATH
+
+DOC_DIRECTORY=".doc"
+PAGES_DIRECTORY="pages"
+FAKER_DIRECTORY="campaigns/data/faker"
+
+# Create .doc directory
+mkdir -p $PROJECT_PATH/$DOC_DIRECTORY
+
+# Function to generate documentation
+function generate_doc() {
+  local JS_DIRECTORY=$1
+  local JS_DIRECTORIES=$(find $JS_DIRECTORY -type d)
+  local JS_FILES=($(find $JS_DIRECTORY -type f -name \*.js))
+
+  # 1 Create directories
+  cd $PROJECT_PATH/$DOC_DIRECTORY
+  mkdir -p $JS_DIRECTORIES
+
+  # 2 Create documentation for each file
+  cd $PROJECT_PATH
+  for JS_FILE in "${JS_FILES[@]}"; do
+    jsdoc2md --no-gfm --files $PROJECT_PATH/$JS_FILE > $PROJECT_PATH/$DOC_DIRECTORY/$JS_FILE.md
```

Comment:
```suggestion
    ./node_modules/.bin/jsdoc2md --no-gfm --files $PROJECT_PATH/$JS_FILE > $PROJECT_PATH/$DOC_DIRECTORY/$JS_FILE.md
```

The binary is no longer global :)

Reply:
done thanks